### PR TITLE
[IMP] sales: add loyalty card button 17

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/odoo/project/3835/tasks/4038367?cids=3

This PR fulfills a request from VPD to add a mention about how there is a Loyalty Card smart button on contact forms in Odoo 17.

an admonition and screenshot have been added to the doc

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @Felicious -- this super quick PR is ready for a Peer Review from you (whenever you get a chance). Thanks! 👍

File: content/applications/sales/sales/products_prices/loyalty_discount.rst [hunk 1]
```diff
@@ -74,6 +74,17 @@ accessible through the smart button located in the upper-right corner of the for
    :align: center
    :alt: Program items smart button on the loyalty program form.
 
+.. note::
+  In Odoo 17 (and above), if a loyalty/coupon card is related to a contact in the database, a
```
Comment (Felicious) [inline content/applications/sales/sales/products_prices/loyalty_discount.rst:78]: totally a nitpick. What you have before works (:
```suggestion
  In Odoo 17 (and later), if a loyalty/coupon card is related to a contact in the database, a
```

**Review by Felicious — APPROVED** [review]
This update looks great, @ksc-odoo !

**Comment by ksc-odoo** [comment]
Thanks for the super helpful suggestion, @Felicious -- I made that adjustment, and believe this one is now ready for Final Review, @StraubCreative (whenever you get a chance). Thanks! 👍

**Comment by ksc-odoo** [comment]
Alrighty, @StraubCreative  I made this already-minor PR *even more* succinct after another look (per our discussion). So, I believe this one is now ready for a Final Review, whenever you get a chance. Thanks! 🙏

File: content/applications/sales/sales/products_prices/loyalty_discount.rst [hunk 2]
```diff
@@ -74,6 +74,16 @@ accessible through the smart button located in the upper-right corner of the for
...
 
+.. note::
+  In Odoo 17 (and later), when a loyalty/coupon card is related to a contact in the database, a
+  :guilabel:`Loyalty Cards` smart button appears on the contact form.
+
+  .. image:: loyalty_discount/loyalty-cards-smart-button.png
+    :align: center
+    :alt: The Loyalty Card smart button as it appears on a contact form in Odoo 17.
+
+  This smart button **only** appears if a loyaly/coupon card is associated with the contact.
```
Comment (StraubCreative) [inline content/applications/sales/sales/products_prices/loyalty_discount.rst:85]: - fix indentation to align with the block's tag
- separate loyalty card and coupon (they have different meanings despite similar functionality)
- remove redundant sentence

```suggestion
.. note::
   In Odoo 17 (and later), when a loyalty card or coupon is associated with a contact in the
   database, a :guilabel:`Loyalty Cards` smart button conditionally appears on the contact form.

   .. image:: loyalty_discount/loyalty-cards-smart-button.png
      :align: center
      :alt: The Loyalty Card smart button as it appears on a contact form in Odoo 17.
```

**Review by StraubCreative — APPROVED** [review]
Approving with comment below @ksc-odoo 
Can tag @samueljlieber after the adjustments are made, thanks!

**Comment by ksc-odoo** [comment]
Thanks for the feedback, @StraubCreative -- Looks like this one is ready for Tech Review, @samueljlieber 👍 Whenever you get a chance... 🙏  Thanks!

File: content/applications/sales/sales/products_prices/loyalty_discount.rst [hunk 3]
```diff
@@ -82,6 +82,16 @@ accessible through the smart button located at the top of the form.
...
 
+.. note::
+   In Odoo 17 (and later), when a loyalty card or coupon is associated with a contact in the
+   database, a :guilabel:`Loyalty Cards` smart button conditionally appears on the contact form.
+
+   .. image:: loyalty_discount/loyalty-cards-smart-button.png
+      :align: center
+      :alt: The Loyalty Card smart button as it appears on a contact form in Odoo 17.
+
+  This smart button **only** appears if a loyaly card or coupon is associated with the contact.
```
Comment (samueljlieber) [inline content/applications/sales/sales/products_prices/loyalty_discount.rst:93]: Alignment and typo :)
```suggestion
   This smart button **only** appears if a loyalty card or coupon is associated with the contact.
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo, thanks for your work on this Sales PR. Looks good to me! Approving with one quick fix :)
..
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**